app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    list_persons = []
+    mari_list = ["wife", "husband"]
+    for humain in people:
+        person = Person(humain["name"], humain["age"])
+        for mari in mari_list:
+            if humain.get(mari) and humain[mari]:
+                setattr(person, mari, humain[mari])
```

Comment:
`wifi` and `husband` must be objects of Person. I think this code in this loop is redundant

Reply:
Without this code, I will not be able to easily find spouses and make links to them !